Makefile

```diff
@@ -357,8 +353,10 @@ container-build: check ## Build containers
 	make docker-build bundle-build
 
 .PHONY: container-push 
-container-push:  ## Push containers (NOTE: catalog can't be build before bundle was pushed)
-	docker-push bundle-push catalog-build catalog-push
+container-push: docker-push bundle-push catalog-build catalog-push must-gather-build must-gather-push ## Push containers (NOTE: catalog can't be build before bundle was pushed)
```

Comment:
good idea to include must-gather 👍🏼 
but can you move must-gather-build to container-build please? catalog-build should be the only exception IMHO, because it needs to run after bundle-push

Reply:
Done.